NDB.Covid19/NDB.Covid19.iOS/Views/DailyNumbers/DailyNumbersViewController.cs

```diff
@@ -40,51 +40,100 @@ public override void ViewDidLoad()
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber2Lbl, StyleUtil.FontType.FontBold, DeathsToday, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber2Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber3Lbl, StyleUtil.FontType.FontBold, TestsConductedToday, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber3Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber4Lbl, StyleUtil.FontType.FontBold, PatientsAdmittedToday, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber4Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber5Lbl, StyleUtil.FontType.FontBold, SmittestopDownloadsTotal, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber5Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber6Lbl, StyleUtil.FontType.FontBold, NumberOfPositiveTestsResultsLast7Days, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber6Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber7Lbl, StyleUtil.FontType.FontBold, PatientsIntensiveCare, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber7Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber9Lbl, StyleUtil.FontType.FontBold, VaccinationsDoseOneToday, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber9Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
+			StyleUtil.InitLabelWithSpacing(DailyNumbersNumber10Lbl, StyleUtil.FontType.FontBold, VaccinationsDoseTwoToday, 1.14, fontMin, fontMax, UITextAlignment.Left);
+			DailyNumbersNumber10Lbl.TextColor = ColorHelper.TEXT_COLOR_ON_PRIMARY;
 
 			// Labels not dependable on device width
-			StyleUtil.InitLabelWithSpacing(DailyNumbersLbl, StyleUtil.FontType.FontBold, DAILY_NUMBERS_HEADER, 1.14, 24, 36);
+			StyleUtil.InitLabelWithSpacing(DailyNumbersTitleOne, StyleUtil.FontType.FontBold, DAILY_NUMBERS_HEADER, 1.14, 30, 36);
+
+			StyleUtil.InitLabelWithSpacing(DailyNumbersLbl, StyleUtil.FontType.FontBold, DAILY_NUMBERS_TITLE_ONE, 1.14, 20, 36);
 			StyleUtil.InitLabelWithSpacing(DailyNumbersOfTheDayTextLbl, StyleUtil.FontType.FontRegular, LastUpdateStringSubHeader, 1.14, 12, 14);
 
-			StyleUtil.InitCenteredLabelWithSpacing(KeyFeature1Lbl, StyleUtil.FontType.FontBold, KEY_FEATURE_ONE_LABEL, 1.14, 16, 18);
-			StyleUtil.InitCenteredLabelWithSpacing(TotalDailyNumbersNumber1Lbl, StyleUtil.FontType.FontRegular, ConfirmedCasesTotal, 1.14, 12, 14);
-			BackgroundView1.BackgroundColor = UIColor.FromRGB(197, 226, 252);
-			BackgroundView1.Layer.CornerRadius = 12;
+			StyleUtil.InitLabelWithSpacing(KeyFeature1Lbl, StyleUtil.FontType.FontRegular, KEY_FEATURE_ONE_LABEL, 1.14, 16, 18, UITextAlignment.Left);
```

Comment:
I think all of these will fail the build, since this method no longer accepts the text alignment parameter. Don't know why it's not failing in pipeline.

Reply:
I have removed the text alignment parameter from method calls.